### PR TITLE
NMS-12723: Events forwarded by Kafka Producer doesn't have any parameters set

### PR DESCRIPTION
**Pull request by cgorantla**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12723

### CODE REVIEW

File: features/kafka/producer/src/main/java/org/opennms/features/kafka/producer/ProtobufMapper.java [hunk 1]
```diff
@@ -220,16 +223,26 @@ public ProtobufMapper(EventConfDao eventConfDao, HwEntityDao hwEntityDao, Sessio
...
-        if (event.getInterface() != null) {
-            builder.setIpAddress(event.getInterface());
+
+        getString(event.getInterface()).ifPresent(builder::setIpAddress);
+
+        for (Parm parm : event.getParmCollection()) {
+            if (parm.getParmName() == null || parm.getValue() == null) {
+                continue;
+            }
+            String value = EventDatabaseConstants.escape(parm.getValue().getContent() == null ? "" :
```
Comment (j-white) [inline features/kafka/producer/src/main/java/org/opennms/features/kafka/producer/ProtobufMapper.java:240]: We don't need to escape here do we?

```suggestion
            String value = parm.getValue().getContent() == null ? "" : parm.getValue().getContent();
```

File: core/daemon/src/main/java/org/opennms/netmgt/daemon/DaemonTools.java [hunk 2]
```diff
@@ -60,7 +60,6 @@ public static void handleReloadEvent(Event e, String daemonName, Consumer<Event>
                 LOG.info("Reload successful.");
 
                 ebldr = new EventBuilder(EventConstants.RELOAD_DAEMON_CONFIG_SUCCESSFUL_UEI, daemonName);
-                ebldr.addParam(EventConstants.PARM_DAEMON_NAME, daemonName);
```
Comment (j-white) [inline core/daemon/src/main/java/org/opennms/netmgt/daemon/DaemonTools.java:63]: Related?

Comment (cgorantla) [reply core/daemon/src/main/java/org/opennms/netmgt/daemon/DaemonTools.java:63]: Not related but saw duplicate params for this event while validating this.

**Comment by cgorantla** [comment]
Except for log/display all the fields are filled for event.
All fields are filled for Alarm.

File: core/daemon/src/main/java/org/opennms/netmgt/daemon/DaemonTools.java (hunk 2 above)
Comment (j-white) [reply core/daemon/src/main/java/org/opennms/netmgt/daemon/DaemonTools.java:63]: Ah I see - it's still added bellow.

**Pull request merged**